### PR TITLE
Spin Wheel

### PR DESCRIPTION
**Pull request by chekoduadarsh**
Added Feature of Spin Wheel for the user to pic one random input.

### CODE REVIEW

**Comment by pnhofmann** [comment]
Hi!

Sorry, but crashes for me
```
~> What can I do for you?
spin wheel
 
welcome to spin the wheel

enter the number of elements in the wheel
5
enter the elements one after another

Some error occurred, please open an issue on github!
Here is error:

Traceback (most recent call last):
  File "/home/phil/git/Jarvis/jarviscli/CmdInterpreter.py", line 223, in try_do
    do(self, s)
  File "/home/phil/git/Jarvis/jarviscli/plugin.py", line 217, in run
    self.get_plugins(command).run(jarvis, new_s)
  File "/home/phil/git/Jarvis/jarviscli/plugin.py", line 209, in run
    self._backend[0](jarvis.get_api(), s)
  File "/home/phil/git/Jarvis/jarviscli/plugins/spinthewheel.py", line 15, in spin
    for i in range(0, num):
TypeError: 'str' object cannot be interpreted as an integer
```

**Comment by chekoduadarsh** [comment]
@pnhofmann sorry for the trouble edited the bug

**Comment by chekoduadarsh** [comment]
I got the following error in travis, can someone help to understand the problem in my code

```

Installing the following packages:

python

By installing you accept licenses for the packages.

[NuGet] Could not connect to the feed specified at 'https://chocolatey.org/api/v2/'. Please verify that the package source (located in the Package Manager Settings) is valid and ensure your network connectivity.

python not installed. An error occurred during installation:

 Unable to resolve dependency 'python3 (= 3.7.4)'.

python package files install completed. Performing other installation steps.

The install of python was NOT successful.

python not installed. An error occurred during installation:

 Unable to resolve dependency 'python3 (= 3.7.4)'.

Chocolatey installed 0/1 packages. 1 packages failed.

 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures

 - python (exited 1) - python not installed. An error occurred during installation:

 Unable to resolve dependency 'python3 (= 3.7.4)'.

The command "choco install python --version 3.7.4" failed and exited with 1 during .

Your build has been stopped.

```

**Comment by pnhofmann** [comment]
> I got the following error in travis, can someone help to understand the problem in my code
> 
> ```
>  Unable to resolve dependency 'python3 (= 3.7.4)'.

https://chocolatey.org/packages/python/3.7.2

"Chocolatey.org is currently offline"

Looks like server issue? In any case nothing you have to worry about ;).

File: jarviscli/plugins/spinthewheel.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from plugin import plugin
+import random
+'''This code picks one of the random inputs given by the user
```
Comment (pnhofmann) [inline jarviscli/plugins/spinthewheel.py]: Maybe make this comment a doc-String: Something like

```
@plugin("Spin wheel")
def spin(jarvis, s):
    """"
    This code picks (...)
    """"
    jarvis.say(...
```

This this text is printed when asked ``help spin wheel``.

File: jarviscli/plugins/spinthewheel.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+like spin wheel'''
+
+
+@plugin("spin wheel")
+def spin(jarvis, s):
+    jarvis.say(' ')
+    jarvis.say('welcome to spin the wheel\n')
+    jarvis.say('enter the number of elements in the wheel')
+    num = jarvis.input()
+    num = int(num)
```
Comment (pnhofmann) [inline jarviscli/plugins/spinthewheel.py]: Crashes if you don't enter a number

```

~> What can I do for you?
spin wheel
 
welcome to spin the wheel

enter the number of elements in the wheel
notanumber
Some error occurred, please open an issue on github!
Here is error:

Traceback (most recent call last):
  File "/home/phil/git/Jarvis/jarviscli/CmdInterpreter.py", line 223, in try_do
    do(self, s)
  File "/home/phil/git/Jarvis/jarviscli/plugin.py", line 217, in run
    self.get_plugins(command).run(jarvis, new_s)
  File "/home/phil/git/Jarvis/jarviscli/plugin.py", line 209, in run
    self._backend[0](jarvis.get_api(), s)
  File "/home/phil/git/Jarvis/jarviscli/plugins/spinthewheel.py", line 13, in spin
    num = int(num)
ValueError: invalid literal for int() with base 10: 'notanumber'
```

Either check this error or (better) use jarvis.input_number
https://github.com/sukeesh/Jarvis/blob/master/doc/API.md#input_number

File: jarviscli/plugins/spinthewheel.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    jarvis.say('enter the elements one after another\n')
+    wheel = []
+    for i in range(0, num):
+        entry = jarvis.input()
+        wheel.append(entry)
+    jarvis.say('Let the wheel spin !!!!!!!!!\n')
+    jarvis.say('result is: ' + random.choice(wheel))
```
Comment (pnhofmann) [inline jarviscli/plugins/spinthewheel.py]: Optional suggestion: Maybe some kind of 're-roll' functionality would be useful?

File: jarviscli/plugins/spinthewheel.py (hunk 1 above)
Comment (chekoduadarsh) [reply jarviscli/plugins/spinthewheel.py]: Thanks and I will look into it

File: jarviscli/plugins/spinthewheel.py (hunk 2 above)
Comment (chekoduadarsh) [reply jarviscli/plugins/spinthewheel.py]: Thanks and I will look into it

**Comment by pnhofmann** [comment]
Looks good! Thanks!

**Pull request merged**